docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -90,3 +90,14 @@ Your Airflow instance likely had specific IAM or Kubernetes permissions that all
 - **Using [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) or [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets)** are the easiest methods for defining permissions.
 
 - **If your Airflow DAGs used [`KubernetesPodOperators`](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/operators.html)**, it's possible that you used the `in_cluster` config or loading a `kube_config` file. When migrating, we recommend switching to [using connections with a `kube_config` JSON blob](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/connections/kubernetes.html) to make things easier.
+
+## Limitations
+
+Currently there are a few limitions of converting airflow DAGs into Dagster Jobs/Assets/Schedules:
+
+- retry-from-failure in dagster is unsupported
+- `prev_execution_date` in airflow templated dags is unsupported
+- `SubDAGOperators` are unsupported
+- Airflow Datasets are unsupported
+
+Most of the current limitations are due to the fact that a locally scoped ephemeral airflow db is created for each run, in the future it will be possible to share this state accross runs.
```

Comment:
```suggestion
The majority of these limitations are due to a locally-scoped, ephemeral Airflow database being created for each run. In future Dagster releases, it will be possible to share this state across runs.
```

Suggesting changes for grammar and clarity